framework/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ Yii Framework 2 Change Log
 2.0.11 under development
 ------------------------
 
+- Enh #12798: Changed `yii\cache\Dependency::getHasChanged()` (to be deprecated in 2.1) to `yii\cache\Dependency::isChanged()` (dynasource)
```

Comment:
For 2.1 line it will be simply "`yii\cache\Dependency::getHasChanged()` was removed" since `yii\cache\Dependency::isChanged()` already exist in 2.0.x line.

Anyway, there is a lot of thing deprecated in 2.0.x and planed to be removed in 2.1.0, but none of them have any information about it in the changelog. Why make an exception here?